plugin/helpers.go

```diff
@@ -37,6 +37,9 @@ type Helpers interface {
 	//
 	// Minimum server version: 5.6
 	KVSetWithExpiryJSON(key string, value interface{}, expireInSeconds int64) error
+
+	// ShouldProcessMessage returns if the message should be process based on the options provided to it.
```

Comment:
Please document the default behaviour

Reply:
@hanzei I will once the implementation is reviewed by all. :) There might other changes as well.